extensions/lifecycle/conductor/LifecycleConductor.js

```diff
@@ -747,12 +747,18 @@ class LifecycleConductor {
                 this.lcConfig.bucketTasksTopic,
                 this.lcConfig.bucketProcessor.groupId, 0, next),
 
-            // check that object tasks topic consumer lag is 0
-            objectTasksCheck:
+            // check that object tasks topic consumer lag for expirations is 0
+            objectTasksExpirationCheck:
             next => this._kafkaBacklogMetrics.checkConsumerLag(
                 this.lcConfig.objectTasksTopic,
                 this.lcConfig.objectProcessor.groupId, 0, next),
 
+            // check that object tasks topic consumer lag for transitions is 0
+            objectTasksTransitionCheck:
```

Comment:
After discussion, it seems this may not be appropriate after all:
* the original goal of checking lag is to ensure we do not start a new listing until the previous one has been terminated, i.e. that each bucket identified by the conductor has been listed → so we should only check the topics which are directly related to listing
* 
* processing expiration is fast, but it may be better to delay li

Reply:
Topics/consumers groups to check identified, but there may be more work needed: moved to https://scality.atlassian.net/browse/BB-449